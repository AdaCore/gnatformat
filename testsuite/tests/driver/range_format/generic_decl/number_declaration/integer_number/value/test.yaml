description: NumberDecl formatting from line selection an integer number declaration.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 26",
    "-EL 26",
    "-EC 36",
    "--pipe",
  ]
