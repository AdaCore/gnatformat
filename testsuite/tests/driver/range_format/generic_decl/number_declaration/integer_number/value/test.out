test.ads(<NumberDecl ["Max"] test.ads:26:4-26:36>) - 26:1-26:36
^
   Max : constant := 500;$

